src/pages/academy/components/video-banner/_DBanner.js

```diff
@@ -68,6 +69,19 @@ const StyledDot = styled.img`
     background: var(--color-grey-17);
     margin: 0 10px 4px;
 `
+const AllVideosButton = styled(LinkButton)`
+    margin: 50px auto -40px;
```

Comment:
I see that there are negative margins here, we usually try to avoid this. 😁  Also, is this a new button configuration from what we usually have in our base component? We can have a discussion about this if it should be added to the base component.